tests/test_device_hardware_simple.py

```
@@ -0,0 +1,218 @@
+        """Test clearing cache when file doesn't exist."""
+        manager = DeviceHardwareManager()
+
+        with patch("os.path.exists", return_value=False):
+            with patch("os.remove") as mock_remove:
+                manager.clear_cache()
+
+                mock_remove.assert_not_called()
+
+
+# Need to import time for timestamp tests
+import time
```

Ruff (0.14.2)

Redefinition of unused `time` from line 9

Remove definition: `time`

(F811)